integrationtests/chrome/chrome_suite_test.go

```diff
@@ -142,7 +142,7 @@ func chromeTestImpl(version protocol.VersionNumber, url string, blockUntilDone f
 	}
 	utils.Infof("Running chrome: %s '%s'", getChromePath(), strings.Join(args, "' '"))
 	command := exec.Command(path, args...)
-	session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
+	session, err := gexec.Start(command, nil, nil)
```

Comment:
I'm still not convinced dropping Chrome logs is the right thing to do, but if you insist…

Reply:
The problem is that Ginkgo dumps the output of the gexec [errWriter](https://godoc.org/github.com/onsi/gomega/gexec#Start) if we run the tests with -v.